### DISCUSSION TITLE
LeanTween conversion

### DISCUSSION THREAD

**Discussion by Eonasdan** [KyryloKuzyk/PrimeTween#16] [General] [answered]
Replacing LeanTween in someone else's code with this. I thought I'd share the comparison code for anyone interested. Thanks for the project.

```cs
LeanTween.rotateLocal([TRANSFORM], [VECTOR3], [DURATION]).setEaseInOutSine();
```

becomes

```cs
 Tween.LocalRotation([TRANSFORM], [VECTOR3], [DURATION], Ease.InOutSine);
```

**Comment by KyryloKuzyk** [comment, marked as answer]
Thanks for sharing! The conversion looks very simple!
